kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/LogReplay.java

```diff
@@ -367,8 +367,7 @@ private Optional<Long> loadLatestTransactionVersion(Engine engine, String applic
 
   /**
    * Loads the domain metadata map, either from CRC info (if available) or from the transaction log.
-   * Note that when loading from CRC info, tombstones (removed domains) are not preserved, while
-   * they are preserved when loading from the transaction log.
+   * Note that when loading from CRC info, tombstones (removed domains) are not preserved.
```

Comment:
No need for this detail, just state this map only contains active domains and any tombstones are not included/are filtered out.

Reply:
Can you please also update the docs for `SnapshotImpl::getDomainMetadataMap` and maybe even `LogReplay::getDomainMetadataMap` (the public API that uses this method here)